src/less/donut-chart/donut-chart.less

```diff
@@ -0,0 +1,185 @@
+    display: block;
+    margin-bottom: 30px;
+    width: 100%;
+}
+.donut-chart__horizontal-metrics {
+    margin-bottom: 20px;
+}
+.donut-chart__metric {
+    display: block;
+    font-size: 30px;
+    font-weight: 700;
+    line-height: 40px;
+}
+.donut-chart__subtitle {
+    display: block;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 18px;
+}
+.donut-chart__segment {
+    opacity: 1;
+    transition: opacity 0.3s;
+}
+.donut-chart__segment--primary {
+    fill: var(--data-viz-primary, var(--color-data-viz-chart-primary));
+    stroke: var(--data-viz-primary, var(--color-data-viz-chart-primary));
+}
+.donut-chart__segment--secondary {
+    fill: var(--data-viz-secondary, var(--color-data-viz-chart-secondary));
+    stroke: var(--data-viz-secondary, var(--color-data-viz-chart-secondary));
+}
+.donut-chart__segment--tertiary {
+    fill: url("#dataVizPattern0");
+    stroke: var(--color-data-viz-chart-quaternary-stroke);
+}
+.donut-chart__segment--quaternary {
+    fill: url("#dataVizPattern1");
+    stroke: var(--color-data-viz-chart-tertiary-stroke);
+}
+.donut-chart__segment--quinary {
+    fill: var(--data-viz-quinary-background, var(--color-data-viz-chart-quinary-background));
+    stroke: var(--data-viz-quinary-stroke, var(--color-data-viz-chart-quinary-stroke));
+}
+.donut-chart--inactive {
+    opacity: 0.2;
+}
+.donut-chart__tooltip {
+    color: var(--color-neutral-7);
+    display: block;
+    font-size: 12px;
+    pointer-events: none;
+    position: fixed;
+}
+.donut-chart__tooltip__label {
+    display: block;
+    font-weight: 700;
+}
+.donut-chart__tooltip__value {
+    display: block;
+}
+.donut-chart__tooltip-container {
+    background: var(--color-neutral-0);
+    border-radius: 8px;
+    display: block;
+    filter: drop-shadow(0 2px 7px var(--color-data-viz-tooltip-shadow-primary))
+        drop-shadow(0 5px 7px var(--color-data-viz-tooltip-shadow-secondary));
+    padding: 8px;
+    position: relative;
+    top: -10px;
+    transform: translate(-50%, -100%);
+    white-space: nowrap;
+}
+.donut-chart__tooltip-container::after {
+    border: 10px solid transparent;
+    border-top: 10px solid var(--color-neutral-0);
+    bottom: 0;
+    content: "";
+    display: block;
+    left: 50%;
+    position: absolute;
+    transform: translate(-50%, 100%);
+}
+.donut-chart__tooltip.donut-chart__tooltip--left .donut-chart__tooltip-container::after {
+    border: 10px solid transparent;
+    border-right: 10px solid var(--color-neutral-0);
+    bottom: 50%;
+    content: "";
+    display: block;
+    left: 1px;
+    position: absolute;
+    transform: translate(-100%, 50%);
+}
+.donut-chart__tooltip.donut-chart__tooltip--right .donut-chart__tooltip-container::after {
+    border: 10px solid transparent;
+    border-left: 10px solid var(--color-neutral-0);
+    bottom: 50%;
+    content: "";
+    display: block;
+    position: absolute;
+    right: 1px;
+    transform: translate(100%, 50%);
```

Comment:
I see a lot of repeated classes here
If we use `BEM` properly, we don't need to repeat these classes.
For the tooltip we can do `class="donut-chart__tooltip donut-chart__tooltip--top` and it should get both class definitions
For BEM, we also shouldn't need to have nested selectors, keep it simple like:
`.donut-chart__tooltip-container::after {`
`.donut-chart__tooltip--top .donut-chart__tooltip-container::after {`


(This might be what Arthur is referring to the simplify)